src/components/avatar/react/Avatar.tsx

```diff
@@ -67,7 +68,9 @@ const DEFAULT_PROPS: IDefaultPropsType = {
  * @return The component.
  */
 const Avatar: React.FC<AvatarProps> = ({
+    actions,
     className = '',
+    hasActions = DEFAULT_PROPS.hasActions,
```

Comment:
Does `hasActions` has to be a props? If you check that there is actions in the `actions` props you would know if there is actions, right?
Also make sure to document the props in the `IAvatarProps` interface in this file

Reply:
I was wondering about that. I looked around the design system and saw that the CommentBlock component has this behavior. I thought maybe it was used to show / hide the actions without changing the "action" props.

Will remove :)